src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,87 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+import mate.jdbc.util.ResetDbUtil;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        System.out.println("Magic STARTS..." + System.lineSeparator());
+
+        //WARNING!!! Only for tests. This reset all schema to initial (empty tables) state.
+        ResetDbUtil.resetDbToInitialState();
+        //WARNING!!!
+
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        driverService.create(new Driver(null, "Abdul", "AA-479974"));
+        driverService.create(new Driver(null, "Hassan", "AH-272727"));
+        driverService.create(new Driver(null, "Mahmud", "AM-386921"));
+
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        manufacturerService.create(new Manufacturer(null, "Bentley", "England"));
+        manufacturerService.create(new Manufacturer(null, "Lexus", "Japan"));
+        manufacturerService.create(new Manufacturer(null, "Mercedes-Benz", "Germany"));
+
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        Car testCarFirst = new Car();
+        testCarFirst.setManufacturer(manufacturerService.get(3L));
+        testCarFirst.setModel("C 300");
+        carService.create(testCarFirst);
+        Car testCarSecond = new Car();
+        testCarSecond.setManufacturer(manufacturerService.get(1L));
+        testCarSecond.setModel("Continental GT Coupe V8");
+        carService.create(testCarSecond);
+        Car testCarThird = new Car();
+        testCarThird.setManufacturer(manufacturerService.get(2L));
+        testCarThird.setModel("Lexus LC Convertible");
+        carService.create(testCarThird);
+
+        carService.addDriverToCar(driverService.get(2L), carService.get(1L));
```

Comment:
relying heavily on magic numbers in main(), see if you can make them constants or retrieve from objects themselves